README.md

```diff
@@ -86,10 +88,10 @@ auto-cpufreq is available on the [snap store](https://snapcraft.io/auto-cpufreq)
 sudo snap install auto-cpufreq
 ```
 
-**Please note:**
+**Please note:** 
```

Comment:
Small changes, but let's remove the empty line at the end

Reply:
done